frontend/src/bundles/common/components/icons/not-found-icon/not-found-icon.tsx

```diff
@@ -0,0 +1,62 @@
+import { type ValueOf } from '~/bundles/common/types/types.js';
+
+const NotFoundIconColor = {
+    PRIMARY: 'primary',
+    SECONDARY: 'secondary',
+} as const;
+
+const NotFoundIconSize = {
+    SMALL: 'sm',
+    MEDIUM: 'md',
+    LARGE: 'lg',
+    EXTRA_LARGE: 'xlg',
+} as const;
```

Comment:
create `enums` folder for enums

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/assets/78347072/a243685f-234c-4e87-a5f8-d52f954a49b8)